wd_helper.au3

```diff
@@ -397,16 +400,13 @@ EndFunc   ;==>_WD_WaitElement
 ; Example .......: No
 ; ===============================================================================================================================
 Func _WD_GetMouseElement($sSession)
+	$_WD_HTTPRESULT = 0 ; Reseting result of last WinHTTP request
```

Comment:
I think the issue to avoid is where the function can potentially exit prior to calling __WD_Post/Delete/Get. If that possibility exists, then the function should perform the reset of $_WD_HTTPRESULT.

Reply:
I agree.

Hope that after last commits:

[core: This function affects $_WD_HTTPRESULT](https://github.com/Danp2/au3WebDriver/pull/247/commits/e0d56ea4d0bedac9fccabc733d50fb9fce3b51a2)
[helper: This function affects $_WD_HTTPRESULT](https://github.com/Danp2/au3WebDriver/pull/247/commits/49ab4d9a1d6e379952cafc475739b7af3f14877b)

that current state, looks fine for you.